src/ducks/tracking/browser.jsx

```diff
@@ -75,6 +75,13 @@ export const useTrackPage = pageName => {
   }, []) // eslint-disable-line react-hooks/exhaustive-deps
 }
 
+export const replaceLastPart = (pageName, newLastPart) => {
```

Comment:
nit: est-ce que ça vaut le coup de faire un petit test ou deux ou on estime que le return est trop simple pour que ce soit pertinent ?

Reply:
I've added an "integration" test that tests tracking events sent from the transaction modal. It tests this part.